src/dfmt/formatter.d

```diff
@@ -850,7 +850,6 @@ private:
             }
             if (sBraceDepth > 0)
                 sBraceDepth--;
-
```

Comment:
Change unrelated to the commit, I would leave the line as it is since it doesn't really changes anything and just pollutes the diff.